storybook nx run problem

storybook problem after install on new workspace 

```
PS C:\Users\conio\Downloads\conio-test> nx run conio:storybook
> nx run octo-clicks-ui:storybook
Cannot find module '@storybook/core/dist/server/build-dev'
Require stack:
- C:\Users\conio\Downloads\conio-test\node_modules\@nrwl\storybook\src\builders\storybook\storybook.impl.js
- C:\Users\conio\Downloads\conio-test\node_modules\@angular-devkit\architect\node\node-modules-architect-host.js
- C:\Users\conio\Downloads\conio-test\node_modules\@angular-devkit\conio\node\index.js
- C:\Users\conio\Downloads\conio-test\node_modules\@nrwl\tao\src\commands\run.js
- C:\Users\conio\Downloads\conio-test\node_modules\@nrwl\tao\index.js
- C:\Users\conio\Downloads\conio-test\node_modules\@nrwl\cli\lib\run-cli.js
```

storybook install inside react libs, any suggestion?

--- a/packages/storybook/src/schematics/init/init.ts
+++ b/packages/storybook/src/schematics/init/init.ts
@@ -5,7 +5,11 @@ import {
   Tree
 } from '@angular-devkit/schematics';
 import { readJsonInTree, updateJsonInTree } from '@nrwl/workspace';
-import { babelLoaderVersion, storybookVersion } from '../../utils/versions';
+import {
+  babelLoaderVersion,
+  babelCoreVersion,
+  storybookVersion
+} from '../../utils/versions';
 import { Schema } from './schema';
 
 function checkDependenciesInstalled(): Rule {
@@ -19,7 +23,7 @@ function checkDependenciesInstalled(): Rule {
       devDependencies['@storybook/react'] = storybookVersion;
       devDependencies['@storybook/addon-knobs'] = storybookVersion;
       devDependencies['babel-loader'] = babelLoaderVersion;
-      devDependencies['@babel/core'] = babelLoaderVersion;
+      devDependencies['@babel/core'] = babelCoreVersion;
     }
     if (
       !packageJson.dependencies['@angular/forms'] &&

